### PR TITLE
115 view cart

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status

**READY
## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.
## Description

A few sentences describing the overall goals of the pull request's commits.

This PR resolves the ViewModel integration on the cart view, allowing for users to view different carts depending on the currently logged in user.
## Resolves Issue Number
#115
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

``` sh
git pull --prune
git checkout 115-view-cart
dotnet run
```
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
1. Load the page
2. Select a user
3. view the user's cart
## Impacted Files in Application

List general components of the application that this PR will affect:
- Cart Controller and View/Cart/Index

### CODE REVIEW

File: Controllers/CartController.cs [hunk 1]
```diff
@@ -117,6 +119,40 @@ public CartController(BangazonContext ctx)
                      new { controller = "Products", action = "Detail", Id = id } ) );
         }
 
+        public async Task<IActionResult> DeleteLineItem([FromRoute]int id)
```
Comment (MattKraatz) [inline Controllers/CartController.cs:122]: Switch to master before you make a new branch :-)

**Review by MattKraatz — APPROVED** [review]
LGTM

**Review by Anulfo — COMMENTED** [review]
LGTM

File: Controllers/CartController.cs (hunk 1 above)
Comment (garrett-vangilder) [reply Controllers/CartController.cs:122]: This was actually intentional because it is a related issue to earlier PR

Comment (MattKraatz) [reply Controllers/CartController.cs:122]: Touche!

**Comment by daynewright** [comment]
LGTM

**Pull request merged**